test/gtest/uct/test_md.cc

```diff
@@ -66,12 +66,18 @@ void* test_md::alloc_thread(void *arg)
 ucs_status_t test_md::reg_mem(unsigned flags, void *address, size_t length,
                               uct_mem_h *memh_p)
 {
+    /* Register memory respecting MD reg_alignment */
+    size_t reg_align  = ucs_max(md_attr().reg_alignment, 1);
+    void *reg_address = address;
```

Comment:
can simply pass length and addr to ucx_align_ptr_range()

Reply:
Right, done